src/GraphQL/Types/QueryArgument.cs

```diff
@@ -48,8 +69,15 @@ public string Name
             }
         }
 
+        /// <summary>
+        /// Gets or sets the description of the argument.
+        /// </summary>
         public string Description { get; set; }
 
+        /// <summary>
+        /// Gets or sets the default value of the argument. Only applicable when the type is not a
```

Comment:
> Only applicable when the type is not a non-null graph type.

IIRC it is wrong. We've already fixed a related bug sometime.

Reply:
Correct, I forgot.